h2o-algos/src/main/java/hex/tree/SharedTreeModel.java

```diff
@@ -622,6 +625,21 @@ public SharedTreeSubgraph getSharedTreeSubgraph(final int tidx, final int cls) {
     return _output._treeKeys[tidx][cls].get().toSharedTreeSubgraph(auxCompressedTree, _output._names, _output._domains);
   }
 
+  /**
+   * Provides a visualization of tree based models when called from Flow.
+   * @param output output file name
+   */
+  @Override
+  public void visualize(String output) {
+    JgraphtPrintMojo jgraphtPrintMojo = new JgraphtPrintMojo();
+    List<SharedTreeSubgraph> subgraphArray = new ArrayList<>();
```

Comment:
confusing variable name, its not an array, would rename to subgraphs, or subgraphList